frontend/src/lib/lemon-ui/LemonCard/LemonCard.tsx

```diff
@@ -6,15 +6,24 @@ export interface LemonCardProps {
     children?: React.ReactNode
     onClick?: () => void
     focused?: boolean
+    'data-attr'?: string
 }
 
-export function LemonCard({ hoverEffect = true, className, children, onClick, focused }: LemonCardProps): JSX.Element {
+export function LemonCard({
+    hoverEffect = true,
+    className,
+    children,
+    onClick,
+    focused,
+    ...props
+}: LemonCardProps): JSX.Element {
```

Comment:
I think we can actually get rid of this component, I don't think it's used elsewhere. I made it because I like the idea of a card with uniform padding but if it's not being used then we should remove. Up to you.

Reply:
I don't mind keeping it. I kept it for the product cards and there's something on the pipelines page that uses it.